src/Build/Instance/ProjectItemInstance.cs

```diff
@@ -1617,7 +1617,7 @@ internal void TranslateWithInterning(ITranslator translator, LookasideStringInte
                         {
                             int key = translator.Reader.ReadInt32();
                             int value = translator.Reader.ReadInt32();
-                            _directMetadata.Set(new ProjectMetadataInstance(interner.GetString(key), interner.GetString(value)));
+                            _directMetadata.Set(new ProjectMetadataInstance(interner.GetString(key), interner.GetString(value), true /* allow built-in metadata names */));
```

Comment:
nit: for new code we generally prefer
```suggestion
                            _directMetadata.Set(new ProjectMetadataInstance(interner.GetString(key), interner.GetString(value), allowItemSpecModifiers: true));
```

Although in this case your comment is clearer than the name!

Reply:
Fixed as suggested, thanks!